templates/vue/cypress/integration/link-authoring.spec.js

```diff
@@ -1,74 +1,114 @@
 describe("Link Authoring", () => {
-  /**
-   * Currently, d3-drag (which handles our drag events) and Cypress are incompatible.
-   * See:
-   *  - https://github.com/cypress-io/cypress/issues/311
-   *  - https://stackoverflow.com/questions/54027884/testing-d3-js-drag-events-with-cypress-js
-   *
-   * We can potentially test this still using Jest and Vue-Testing-Lib:
-   *  - https://github.com/wynnset/tapestry-wp/pull/767
-   */
-  describe.skip("Linking", () => {
-    it(`
-      Given: Two unlinked leaf nodes
-      When: The add button is dragged from one to the other
-      Then: A confirmation should appear that, when confirmed, adds the link
-    `)
-
-    it(`
-      Given: The link confirmation modal
-      When: The cancel button is pressed
-      Then: The link should not be created
-    `)
-
-    it(`
-      Given: Two linked nodes
-      When: A link is added
-      Then: The link should not be created
-    `)
-
-    it(`
-      Given: Two nodes, one without add permission
-      When: A link is added
-      Then: An alert should appear and the link not created
-    `)
-
-    it(`
-      Given: An accordion node and a regular node
-      When: A link is added from the accordion to the node
-      Then: A confirmation should appear that, when confirmed, adds the node as a row
-    `)
-
-    it(`
-      Given: An accordion node and a regular node
-      When: A link is added from the accordion to the node
-      Then: A confirmation should appear that, when cancelled, adds only the link
-    `)
+  beforeEach(() => {
+    cy.fixture("stump.json").as("stump")
+    cy.setup("@stump")
```

Comment:
Could we name this something less generic than `stump` in case in the future we want to add other types of tapestries? Something like `base-tapestry` or something?

Reply:
There's already other types of tapestries under `fixtures`, I think this `stump` name refers to the fact it's a root node with two nodes so it's like a  "mini-tree" 😄

Feel free to rename it to something more appropriate though.